r/R/dplyr-eval.R

```diff
@@ -231,7 +239,6 @@ abandon_ship <- function(err, env) {
   }
 
   # Else, warn, collect(), and run in regular dplyr
-  call <- get("call", envir = env)
```

Comment:
I'm a little surprised by this deletion, was this essentially overwritting what was in `call` when it ran? Or something else?

Reply:
I just moved it up to L227

Reply:
AAAAAH I totally missed that. Thanks